libs/sdk-ui-kit/src/Header/generateHeaderMenuItemsGroups.ts

```diff
@@ -0,0 +1,111 @@
+// (C) 2007-2021 GoodData Corporation
+import { ISettings, IWorkspacePermissions } from "@gooddata/sdk-backend-spi";
+import { IHeaderMenuItem } from "./Header";
+import { isFreemiumEdition, shouldHidePPExperience } from "../utils/featureFlags";
+
+/**
+ * @internal
+ */
+export function generateHeaderMenuItemsGroups(
+    featureFlags: ISettings,
+    workspacePermissions: IWorkspacePermissions,
+    workspaceId?: string, // parsed from bootstrapResource.current.project.links.self
```

Comment:
These comments make only sense in bear, for other backends this data will come from other places. Please remove those comments

Reply:
I vemote these comments thanks